### DISCUSSION TITLE
Music rewinding triangles down

### DISCUSSION THREAD

**Discussion by ghost** [ppy/osu#20746] [Ideas] [answered]
Currently the triangles in the home menu logo just move up constantly and faster when sliding the music players "bar" (forgot the name)
![image](https://user-images.githubusercontent.com/71771134/195614340-55f4e445-ade3-4f6c-8540-e1e75ba2fc65.png)
I think it would be cool if rewinding would make the triangles go DOWN. This idea is 100% useless :)

**Comment by WindowsMeosu** [comment, marked as answer]
so basically rewinding = triangle animation on the logo results in going down instead of going up?! this seems pretty straight-forward but also pretty unnecessary since it's just adding slight detail to the animation. I see your point, but I don't know about this here.

**Reply by ghost** [reply]
Attention to detail is nice
> but I don't know about this here.

Should I post it somewhere else :)?

**Reply by WindowsMeosu** [reply]
as in "I don't know about this here" I was referring to the fact that it just didn't seem right to make the cut. it shouldn't hurt anything, so it seems fine.

**Reply by mk56-spn** [reply]
its probably not something someones going to go out of their way to add anytime soon but assuming core devs arent explicitly against it theres always a chance someone will pick it up and do it. theres no "cut" per say

**Reply by peppy** [reply]
I'd have to see it in action. No guarantees I'd like it.

Also implementation will not be super simple due to how triangles works. Would probably need to rewrite that class.
